### ISSUE TITLE
[Feature] Page read control and size request

### ISSUE THREAD

**Issue by echofire26** [manga-download/hakuneko#4275]
First thank you for HakuNeko.

As to the point of the feature request, Was hoping it might be possible to update the reader functionality, or offer a different navigation format along with a image resize request similar to #3773.  The current reader format is dependent on the scroll functionality, my request is to offer individual pages with "fit to screen option" and a left right arrow key (or preferred bind key) navigation.  This would be inline with many modern readers and give a more book feel to the reader as is.

Thank you.

**Comment by Sheepux** [comment]
did you try * and / and also the "magic scroll" (spacebar) ?

also a new reader is being worked in haruneko (can already be tested on the other github)

**Comment by echofire26** [comment]
Yes, I tried the * feature I will try space bar.  Based on description though it's not what I'm referring to.  I will check out the GitHub option you are referring too. 

My request is specifically turning the reader into a reader rather than an image scroller where the page is maximzed to fit the monitor (vertical typically) while keeping aspect ratio.

**Comment by Sheepux** [comment]
ok, so the request is a "fit to height" just like * is fit to width. Understood

**Comment by echofire26** [comment]
Thank you.

**Comment by vegaPunk555** [comment]
I have the same problem, please give me advice on how to fix 
![2024-03-28 21_57_28-Window](https://github.com/manga-download/hakuneko/assets/139472144/3b34e366-3bcb-45db-ac70-36ace2d5da6d)

**Comment by Sheepux** [comment]
"Same problem" ? It's not a problem, it's a feature request, how is your screenshot related to fit to height ?

**Comment by vegaPunk555** [comment]
Brother, I need all the pages to be the same height and width, how do I do it?
![2024-03-30 21_38_58-Settings](https://github.com/manga-download/hakuneko/assets/139472144/0a34b084-31f1-4764-b98a-9211c6326d60)

**Comment by Sheepux** [comment]
There is no such feature  in hakuneko.

**Comment by vegaPunk555** [comment]
Bro IDK if it's a feature, setting or a bug how do I get it all to the same size in PDF format?

**Comment by echofire26** [comment]
Seriously, you need to drop it.  I put this in as a feature request.  It's something that would be nice to have, and I think it would really improve the reading experience.  As it stands though, the software does not have the functionality.  Let's just hope the devs can institute such a feature in the future and thank them in the meantime for even providing such software.

**Comment by Sheepux** [comment]
@echofire26 the fit to height in the reader is almost there in haru. I need to polish a few other things in the reader and it will be easier to embed it. (adding this issue to the haru nice to have list)

**Comment by echofire26** [comment]
@Sheepux Thank you!  I look forward to its release, and thanks again for all that y'all do and have done.
